airflow/scripts/git_scripts/aggregate_user_activty.py

```diff
@@ -0,0 +1,94 @@
+
+import requests
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+
+token = 'put_token_here'   # add your token
+
+headers = {'Authorization': f'token {token}', 'Accept': 'application/vnd.github.v3+json'}
+
+# Define the date range
+since = (datetime.now() - timedelta(days=7)).isoformat()
+until = datetime.now().isoformat()
+
+
+def fetch_data(url, params=None):
+    """Fetch data from GitHub API"""
```

Comment:
You can remove the notes. If the function is short and self-explanatory, notes are unnecessary.

Reply:
Deleted notes from short functions